src/util/urls.js

```diff
@@ -1,4 +1,6 @@
 const urls = {
+  agreementView: id => `/erm/agreements/${id}`,
```

Comment:
I'd like to do this through the Registry please, take a look at the documentation here: https://wiki.folio.org/display/ERM/Registry and see if you can work out how to get hold of the agreement lookup function